auv_setup/launch/auv.launch

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<launch>
+  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}" />
+ 
+  <!-- Input arguments -->
+  <arg name="type" default="real"/> <!-- real|simulator -->
```

Comment:
It may be beneficial to change the default argument to 'simulator' for arg 'type' in auv_setup/launch/auv.launch, as most of the full-system testing at this point in time is going to be simulator based.

Reply:
I agree with that; is addressed in commit ced9b52